metapool/metapool.py

```diff
@@ -370,7 +371,7 @@ def read_plate_map_csv(f, sep='\t', qiita_oauth2_conf_fp=None):
 
 # method to read minipico output
 def read_pico_csv(f, sep='\t', plate_reader='SpectraMax_i3x',
-                  min_conc=0, max_conc=150,
+                  min_conc=0, max_conc=250,
```

Comment:
It would be helpful to have a comment about where the max_conc=250 hardcoding came from, and under what circumstances it might change.